dockerfiles/artifactory/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM node:8-slim
```

Comment:
Is there a reason this is on node 8 rather than 12 like the others?

Reply:
It was stupidly copied from here😱😱😱

https://github.com/snyk/broker/blob/master/dockerfiles/artifactory-cr/Dockerfile